scripts/update-dependencies-config.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+module.exports = {
+  electron: [
+    '@electron/remote',
+    '@electron/rebuild',
+    'browserslist',
+    // NB: We're always trying to update to latest major, this usually implies
+    // breaking changes, but those rarely affect us. If it becomes a problem, we
+    // can always change this code to lock it to whatever major version of
+    // electron compass is currently at
+    'electron',
```

Comment:
This is a "breaking change" compared to what the old script was doing, manually trying this update locally it doesn't seem to break anything and as mentioned I do think we rarely affected by those, so it's probably okay to try to always update to latest, but tell me if you have thoughts on that.